app/views/layers/_form.html.haml

```diff
@@ -90,30 +90,38 @@
         %hr
         %div.compact
           .grid-x.grid-padding-x
-            .large-6.medium-6.small-12.cell
+            .large-12.medium-12.small-12.cell
+              %label Predefined Basemaps
+              %select#basemaps
+                %option{value: '', "data-url": '', "data-backgroundcolor": '', "data-attribution": '' }
+                - basemaps.each_value do |b|
+                  %option{value: b['id'], "data-url": b['url'], "data-backgroundcolor": b['backgroundcolor'], "data-attribution": b['attribution'] }
+                    = b['title']
+              %p.shy
+                Please select one the prefinied basemaps
```

Comment:
typo:   Please select one **of** the prefinied basemaps

Reply:
Please select one of the pre**de**finied basemaps

Reply:
done! thanx